### ISSUE TITLE
ChatGPT_answear faild

### ISSUE THREAD

**Issue by fcjasma** [The-CodingSloth/kahoot-god#7]
i'm sorry for asking but i was struggling about 5 hours to make this code work AND finally something is happening. I can run code and it reads questions etc. but in the end show message like in title. I've read in other issue that it may be caused because I try to use free API key. So again sorry for asking but is there any way to check if it finally works without spending 5$ on that key?

**Comment by bean-frog** [comment]
Yes, you need to purchase an OpenAI API key. Trust me, if you use gpt-3.5-turbo that $5 will go a long way. According to OpenAI's pricing page, the input cost for 1000 tokens is $0.0015 and the output cost for the same amount is $0.002. So with this program you will get a very worthwhile amount of use from your $5

**Comment by oavaldezi** [comment]
I've placed my OpenAI API key in line 7, enclosed in double quotes after openai.api_key = , and I've checked that I have a balance with OpenAI, but I keep getting "chatGPT_answer failed"

As suggested elsewhere, I replaced this (lines 78-79):

```
except:
    print("chatGPT_answer failed")

```
with this

```
except Exception as e:
    print(f"chatGPT_answer failed with error: {str(e)}")
```

and the error now is: "chatGPT_answer failed with error: cannot access local variable 'response' where it is not associated with a value"
